src/clims/api/serializers/models/task.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import
+
+from rest_framework import serializers
+
+
+class TaskSerializer(serializers.Serializer):
+    id = serializers.CharField()
+    tracked_object = serializers.SerializerMethodField()
```

Comment:
Right. This is something we will have to document.

Camunda has the concept of adding a "business key" to a process. When you start a process, you can assign a business key to it, which is just a process variable, but it's a special one, in that you can query for it specifically and I assume it's optimized to query for it in some cases.

The idea is that this "business key" is what the process instance is handling. I came up with the term `tracked_object` to refer to this object, as it's tracked through the whole workflow.

In our context, the `tracked_object` is always a `Substance` currently, but in the future it might for example be a `Container`. If it's a substance, the business key of the tracked object will be `Subtance/<guid>`.